src/Forms/NumericField.php

```diff
@@ -101,10 +101,24 @@ protected function getNumberType()
         return NumberFormatter::TYPE_DOUBLE;
     }
 
+    /**
+     * In some cases and locales, validation expects non-breaking spaces.
+     * This homogenises narrow and regular NBSPs to a regular space character
+     *
+     * @param string|null $value
+     * @return string
+     */
+    private function clean($value): string
+    {
+        return trim(str_replace([
+            "\u{202F}", "\u{2009}"
+        ], ' ', $value ?? ''));
```

Comment:
Please format this in one of these two ways to make it more readable:
```suggestion
        return trim(str_replace(["\u{202F}", "\u{2009}"], ' ', $value ?? ''));
```
or
```suggestion
        return trim(str_replace([
            "\u{202F}",
            "\u{2009}",
        ], ' ', $value ?? ''));
```